library/CMakeLists.txt

```diff
@@ -214,3 +228,29 @@ add_custom_target(lib DEPENDS mbedcrypto mbedx509 mbedtls)
 if(USE_STATIC_MBEDTLS_LIBRARY AND USE_SHARED_MBEDTLS_LIBRARY)
     add_dependencies(lib mbedcrypto_static mbedx509_static mbedtls_static)
 endif()
+
+include(PkgBuildConfig)
```

Comment:
CI says (with CMake 3.5.1, note that we support all versions of CMake ≥ 2.8.12):
```
CMake Error at ../../../library/CMakeLists.txt:232 (include):
  include could not find load file:

    PkgBuildConfig


CMake Error at ../../../library/CMakeLists.txt:236 (pkg_build_config):

  Unknown CMake command "pkg_build_config".


-- Configuring incomplete, errors occurred!
```